builds/e2e/longhaul.yaml

```diff
@@ -168,79 +104,76 @@ jobs:
         - Agent.OSArchitecture -equals ARM
         - agent-osbits -equals 32
         - run-long-haul -equals true
-    variables:
-      edgelet.artifact.name: 'iotedged-debian9-arm32v7'
-      aziotis.artifact.name: 'packages_debian-9-slim_arm32v7'
-      aziotis.package.filter: 'aziot-identity-service_*_armhf.deb'
+        - client-transport -equals amqp
```

Comment:
Stupid question, why "client-transport -equals amqp" in demands, and also set "clientModuleTransportType: 'amqp'"?

Reply:
We need to make sure the following case doesn't happen:
1. 1 longhaul arm agent available
2. agent starts running amqp job and successfully deploys
3. same agent gets picked up for mqtt job and successfully deploys

This results in the amqp job not running, as it was overwritten.